.gitignore

```diff
@@ -6,3 +6,5 @@ target
 perf.data*
 
 Cargo.lock
+
+lindera-ipadic-*.tar.bz2
```

Comment:
Should we add lindera-ipadic too? Or we need dict binary for purpose to run cargo build?

Reply:
Yes, it is necessary.  When cargo creates a package that publishes a crate to crates.io, the cargo requires all the resources needed to build the crate.  If it is excluded by .gitignore, the resources required for the build cannot be passed to the cargo.

Reply:
Yes, it is necessary.  When cargo creates a package that publishes a crate to crates.io, the cargo requires all the resources needed to build the crate.  If it is excluded by .gitignore, the resources required for the build cannot be passed to the cargo.

Reply:
I see, thanks!